media/css/newsletter/newsletter-developer.scss

```diff
@@ -5,26 +5,40 @@
 @import '../pebbles/includes/lib';
 
 .section-intro {
-    background: url('/media/img/newsletter/developer/heart.svg') top 30px center no-repeat,
-                url('/media/img/newsletter/developer/grid.svg') top left repeat,
-                #0c99d5;
-    @include background-size(200px 199px, 60px 60px);
-    height: 230px;
+    background: url('/media/img/newsletter/developer/hearts.svg') top 30px center no-repeat,
+                url('/media/img/newsletter/developer/bg-image.png') top left no-repeat,
```

Comment:
You've changed the names to lowercase here but also need to rename the files to match. These two images are a 404 right now. It's possible you renamed them locally but they didn't get committed?

Reply:
That is exactly what happened! _facepalm_